OrbitCore/LinuxPerfEventProcessor.cpp

```diff
@@ -12,16 +12,8 @@
 
 #include "Profiling.h"
 
-void LinuxPerfEventProcessor::PushEvent(int origin_fd,
-                                        std::unique_ptr<LinuxPerfEvent> event) {
-#ifndef NDEBUG
-  if (last_processed_timestamp_ > 0 &&
-      event->Timestamp() <
-          last_processed_timestamp_ - PROCESSING_DELAY_MS * 1'000'000) {
-    PRINT("Error: processed an event out of order.\n");
-  }
-#endif
-
+void PerfEventQueue::PushEvent(int origin_fd,
```

Comment:
Move PerfEventQueue implementation to a PerfEvenQueue.cpp?

Reply:
Isn't it clearer for now to keep the two classes in the same file as they work tightly together?

Reply:
maybe clearer from navigation point of view to keep every class in a separate header but I do not have strong feelings one way or the other.